tasks/cephfs/test_dump_tree.py

```diff
@@ -0,0 +1,46 @@
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+import random
+import os
+
+class TestDumpTree(CephFSTestCase):
+    def get_paths_to_ino(self):
+        inos = {}
+        p = self.mount_a.run_shell(["find", "./"])
+        paths = p.stdout.getvalue().strip().split()
+        for path in paths:
+            inos[path] = self.mount_a.path_to_ino(path, False)
+
+        return inos
+
+    def populate(self):
+        self.mount_a.run_shell(["git", "clone",
+                                "https://github.com/ceph/ceph-qa-suite"])
+        
+    def test_dump_tree(self):
```

Comment:
Hmm, why do we need a lot of files and then to pick random ones from within that?  Putting randomness into tests feels like it might be asking for trouble.

What if you created these paths:
```
alpha1/
alpha1/bravo
alpha1/bravo/charlie
alpha2/
```
Then test that dump on / includes on everything, dump on alpha1/ with depth high excludes alpha2, and dump on alpha1 with depth=1 excludes charlie.  Would that be missing any cases?